### ISSUE TITLE
Update UI text to refer to Mapeo not OpenStreetMap

### ISSUE THREAD

**Issue by gmaclennan** [digidem/mapeo-desktop#94]
Current text is around OSM / internet, needs to be clear it is for communities

Should change text in at least English and Spanish

- [ ] Intro
- [ ] Upload

**Comment by karissa** [comment]
This should be fixed now with the latest 1.11.2 release

**Comment by noffle** [comment]
:tada:!!!

**Comment by aliya-ryan** [comment]
The version 4.0.4-beta shows osm text when you go to save data from the editor view, it says 'save to OSM' and then when you hit save it says 'uploading to osm' or something like that. I've had 2 people query this in last couple of days to me - can we remove / change text?

**Comment by noffle** [comment]
@aliya-ryan here is what I see when I hit `Save`, is it different from what you see?

![save](https://user-images.githubusercontent.com/489362/61752304-0a225a80-ad60-11e9-9e4d-9cba3c654aa3.png)

When I hit 'Save' I see "Saving changes" with a spinner.

**Comment by noffle** [comment]
test

**Comment by aliya-ryan** [comment]
Hey - yes different to what I see: 
<img width="1137" alt="Screenshot 2019-07-24 at 09 16 03" src="https://user-images.githubusercontent.com/21277147/61776966-fa545600-adf3-11e9-8409-aff10a664f6b.png">
<img width="1084" alt="Screenshot 2019-07-24 at 09 16 12" src="https://user-images.githubusercontent.com/21277147/61776967-fa545600-adf3-11e9-96cf-45c706b04bd5.png">
<img width="709" alt="Screenshot 2019-07-24 at 09 16 39" src="https://user-images.githubusercontent.com/21277147/61776968-faecec80-adf3-11e9-8f71-e349f934f3b9.png">

**Comment by noffle** [comment]
It looks like this text gets pulled from [iD-mapeo](https://github.com/digidem/iD-mapeo) (our fork of the iD editor), and that @karissa changed this text in [a3c1ac7ab1d2dac9a45cd580c9e597b649970d5c](https://github.com/digidem/iD-mapeo/commit/a3c1ac7ab1d2dac9a45cd580c9e597b649970d5c).

**However**, only `en` was changed, and I'm guessing your locale is `en-GB`, which wasn't updated.

I just edited our `iD-mapeo` fork so that our `en` and `en-GB` monkey patches are the same, and published it. It will be included in the next Mapeo Desktop release.
